src/hx711.rs

```diff
@@ -212,6 +212,10 @@ where
         Ok(raw as f32 * self.scale)
     }
 
+    fn set_offset(&mut self, offset: i32) {
```

Comment:
Will you make this type
```
    fn set_offset(&mut self, offset: Self::Offset) {
```

Reply:
Oh, I didn't notice that type. Yes, it should be used in `set_offset`.

By the way, I tested `set_offset`. It works.

This is `TARE_OFFSET` (stored in RTC RAM) before deep sleep: 

```
I (574) heavy_cloud: TARE_OFFSET before check: 0
```

This is after:

```
W (573) heavy_cloud: TARE_OFFSET before check: -70599
```